docs/userguide/constraints.md

```diff
@@ -222,10 +233,123 @@ DiscreteLinkedParametersConstraint(
+
+|   | Solvent 1    | Solvent 2    | Solvent 3    |
+|---|--------------|--------------|--------------|
+| 1 | Substance 43 | Substance 3  | Substance 12 |
+| 2 | Substance 43 | Substance 12 | Substance 3  |
+| 3 | Substance 3  | Substance 12 | Substance 43 |
+| 4 | Substance 3  | Substance 43 | Substance 12 |
+| 5 | Substance 12 | Substance 43 | Substance 3  |
+| 6 | Substance 12 | Substance 3  | Substance 43 |
+
+```{note}
+Complex properties such as permutation invariance not only affect the search space, but 
+should ideally also constrain the surrogate model. For instance, the kernels in a 
+Gaussian process can be made permutation invariant to reflect this constraint, which 
+generally results in a better learning curve. Note, however, that not all surrogate 
+models provided by BayBE can easlily include such invariances. In these cases the 
+invariance is silently ignored, which will not be an issue, but just means these models 
+cannot benefit from a priori known constraints and invariances between parameters.
+```
+
+Let's add to the mixture example the fact that not only the choice of substance but also 
+their relative mixture factions are parameters "Fraction 1", "Fraction 2" and 
+"Fraction 3".
+This example also implies that the solvent parameters depend on their corresponding 
+fraction being ``> 0.0``, because in the case ``== 0.0`` the choice of solvent is 
+irrelevant. This models a campaign that allows "up to, but not necessarily, 
+three solvents".
+
+```{important}
+If some of the ``parameters`` of the ``DiscretePermutationInvarianceConstraint`` are 
+dependent on other parameters, we require that the dependencies are provided as a 
+``DiscreteDependenciesConstraint`` to the ``dependencies`` argument of the 
+``DiscretePermutationInvarianceConstraint``. This 
+``DiscreteDependenciesConstraint`` will not count towards the maximum limit of one 
+``DiscreteDependenciesConstraint`` discussed [here](#DDC).
+```
+
+The ``DiscretePermutationInvarianceConstraint`` below applies to our example and 
+removes permutation invariant combinations of solvents that have additional 
+dependencies as well:
+
+```python
+from baybe.constraints import DiscretePermutationInvarianceConstraint, DiscreteDependenciesConstraint, ThresholdCondition
+
+DiscretePermutationInvarianceConstraint(
+    parameters=["Solvent 1", "Solvent 2", "Solvent 3"],
+    # `dependencies` is optional, it is only required if some of the permutation 
+    # invariant entries in `parameters` have dependencies on other parameters
+    dependencies=DiscreteDependenciesConstraint(
+        parameters=["Fraction 1", "Fraction 2", "Fraction 3"],
+        conditions=[
+            ThresholdCondition(threshold=0.0, operator=">"),
+            ThresholdCondition(threshold=0.0, operator=">"),
+            ThresholdCondition(threshold=0.0, operator=">"),
+        ],
+        affected_parameters=[["Solvent 1"], ["Solvent 2"], ["Solvent 3"]],
+    ),
+)
+```
+
```

Comment:
At the end of this explanation, I (as a user) do not fully understand what is being modeled here and how it is modeled: From my understanding, this only makes sense if Solvents 1,2 and 3 can all have exactly the same values (e.g. "Clean Water", "Ethanol", "Salty Water"), they all need to be (strictly?) larger than 0 and I do not care about the order: If I have e.g. a mixture containing 1/3 of each of the substances, I do not care if Solvent 1 describes the clean water, the salty water, or the ethanol.
Is this interpretation correct? If not, we should adjust the description accordingly. Also, a short summary/explanation like that, i.e., with a specific example might be helpful (but I leave that up to you whether you think this might be beneficial)

Reply:
youre confusing me Alex
Your summary interpretation is correct which would genreally mean the explanation is perfect :D 

So what should I change?

Reply:
That was exactly the point of my comment :D I wanted to verify that the interpretation that I had after reading is is what you intended it to be. This seems to be the case, so no need to change!